index.html

```diff
@@ -8,34 +8,204 @@
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </article>
+                <div class="languages">
+                    <div class="lang1"><img src="image/Tag_html.svg" alt="HTML Tag"></div>
+                    <div class="lang2"><img src="image/Tag_css.svg" alt="HTML Tag"></div>
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
+                <div class="nav-button">
+                    <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Enabled.svg" alt="See Project button"></a> 
+                </div>    
+            </div>
+        </section>
+
+        <section>
+        <div class="container_6"> 
+                <div class="header_6">
+                    <img src="image/Primary Text.svg" alt="Javascript">
+                </div>    
+            <p class="content_6">Hello I’m a software developer! I can help you build a product , feature or website
+                     Look through some of my work and experience! If you like what you see and have a 
+                     project you need coded, don’t hestiate to contact me.</p>
+            <div class="connect"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Social Media.svg" alt="Let's Connect"></a></div>
+            <div class="social">
+                <ul>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                    <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                </ul>
+            </div>
+            <div class="nav-button">
+                <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Resume.svg" alt="Get my Resume"></a> 
+            </div> 
```

Comment:
- [ ] Kindly change the image used as a button and use a button tag with its text based on the template you choose.